src/Simulation/QCTraceSimulator.Tests/PrimitiveOperationsCounterTest.cs

```diff
@@ -120,6 +120,30 @@ void TDepthTest()
             Assert.Equal(1, sim.GetOperationMetric<TDepthOne>(MetricsNames.DepthCounter.Depth));
         }
 
+        [Fact]
+        void TDepthAndWidthTest()
+        {
+            TracerSimulator sim = null;
+            sim = RunDepthWidth(optimizeDepth: true);
+            Assert.Equal(4, sim.GetOperationMetric<TDepthWidth>("Width"));
+            Assert.Equal(2, sim.GetOperationMetric<TDepthWidth>("Depth"));
+            sim = RunDepthWidth(optimizeDepth: false);
+            Assert.Equal(3, sim.GetOperationMetric<TDepthWidth>("Width"));
+            Assert.Equal(3, sim.GetOperationMetric<TDepthWidth>("Depth"));
+        }
+
+        TracerSimulator RunDepthWidth(bool optimizeDepth) {
+            NewTracerConfiguration config = new NewTracerConfiguration();
+            config.UseDepthCounter = true;
+            config.OptimizeDepth = optimizeDepth;
+            config.TraceGateTimes = NewTraceGateTimes.ControlledZOnly;
```

Comment:
```suggestion
            var config = new NewTracerConfiguration()
            {
                UseDepthCounter = true,
                OptimizeDepth = optimizeDepth,
                TraceGateTimes = NewTraceGateTimes.ControlledZOnly
            };
```

Reply:
Updated code to use "simplified" initialization. Such initialization typically prevents line-by-line debugging and makes quoting portions of code harder, but this isn't a significant drawback here.
Explicit type is still used - easier to understand. When looking up variable, less time and mental effort required to figure out its type.